api/client.proto

```diff
@@ -56,22 +56,26 @@ message Client {
   // If the authorization request does not pass a redirect URI, the first one
   // from this list is taken.
   repeated string redirect_uris = 9 [(gogoproto.customname) = "RedirectURIs"];
+  // The allowed logout redirect URIs against which client initiated logout
+  // requests are checked. If the authorization request does not pass a redirect
+  // URI, the first one from this list is taken.
+  repeated string logout_redirect_uris = 10 [(gogoproto.customname) = "LogoutRedirectURIs"];
```

Comment:
Changing the numbers of existing fields breaks API. Use a unused number for new fields.